README.md

```diff
@@ -356,6 +358,28 @@ You may want to know which delta configuration values the emulation mode has sel
 
 The within-line highlighting rules employed by diff-highlight (and therefore by diff-so-fancy) are deliberately simpler than Delta's Levenshtein-type edit inference algorithm (see discussion in the [diff-highlight README](https://github.com/git/git/tree/master/contrib/diff-highlight)). diff-highlight's rules could be added to delta as an alternative highlighting algorithm, but that hasn't been done yet.
 
+### `--color-moved` support
+
+[_**Unreleased feature**: available now if you build Delta from source, and will be included in the next Delta release. See [#72](https://github.com/dandavison/delta/issues/2)._]
```

Comment:
@dandavison there's a typo in the URL :) it says "2" when it should say "72".

Reply:
Thank you!